src/services/students_service.py

```diff
@@ -34,7 +40,9 @@ def add_attendance(self, cpf_key: str):
         current_class = self.schedule.get_current_class()
         if current_class is None:
             raise NotOngoingLesson(
-                "Não há aula em andamento. As presenças só podem ser registradas nos intervalo entre 17:45 até 20:00 e 20:15 até 22:00"
+                "Não há aula em andamento. As presenças só podem ser registradas"
+                + f" nos intervalo entre {self.first_half_begin} até {self.first_half_end}" 
+                + f" e {self.second_half_begin} até {self.second_half_end}"
```

Comment:
```suggestion
                f"Não há aula em andamento. As presenças só podem ser registradas
                  nos intervalo entre {self.first_half_begin} até {self.first_half_end} 
                  e {self.second_half_begin} até {self.second_half_end}"
```

Acho melhor fazer desse jeito do que ficar concatenando varias strings

Reply:
@vianaz Eu fiz isso porque facilita a leitura, sem concatenar fica uma string bastante longa. Fiz a mudança!

Reply:
@maecco se eu não me engano cê pode usar \, para fazer uma quebra linha no código mas que não afeta output, mas enfim, apenas comentando mesmo, vou resolver a thread :))